dev-util/jenkins-bin/files/jenkins-bin-r2.service

```diff
@@ -3,9 +3,18 @@ Description=Jenkins Daemon
 
 [Service]
 EnvironmentFile=/etc/conf.d/jenkins
-Type=forking
-ExecStart=/usr/bin/java ${JENKINS_JAVA_OPTIONS} -DJENKINS_HOME=${JENKINS_HOME} -jar ${JENKINS_WAR} --daemon --logfile=/var/log/jenkins/jenkins.log --httpPort=${JENKINS_PORT} --debug=${JENKINS_DEBUG_LEVEL} --handlerCountMax=100 --handlerCountMaxIdle=20 --accessLoggerClassName=winstone.accesslog.SimpleAccessLogger --simpleAccessLogger.format=combined --simpleAccessLogger.file=/var/log/jenkins/access_log ${JENKINS_ARGS}
 User=jenkins
+Type=simple
```

Comment:
This change has already been handled via the associated bug. If the other changes are still needed then they will need to be redone based on the r4 version of the service file.